### DISCUSSION TITLE
needed help for plots

### DISCUSSION THREAD

**Discussion by basunjoy3849** [Chia-Network/chia-blockchain#6837] [Support]
all my plots are showing
![Screenshot 2021-06-18 001832](https://user-images.githubusercontent.com/83465336/122456118-cf1c1680-cfca-11eb-8188-6896fdac9f52.png)
 this what to do

**Comment by loppefaaret** [comment]
if this is a normal setup, and not a remote harvester, first try and reboot
if the issue still persist, try this:
COMPLETE CONFIG RESET

Close down the Chia app, and open up a PowerShell window, and execute the below commands one at a time:
```
mv ~\.chia\mainnet ~\.chia\old
cd ~\AppData\Local\chia-blockchain\app-1.1.7\resources\app.asar.unpacked\daemon\
.\chia.exe init
mv ~\.chia\old\db ~\.chia\mainnet\
mv ~\.chia\old\wallet ~\.chia\mainnet\
```
then try and open up Chia app again

**Comment by BrandtH22** [comment]
@basunjoy3849 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
